src/Tizen.NUI/src/public/NUIApplication.cs

```diff
@@ -383,7 +383,6 @@ static public void PreLoad()
         {
             Interop.Application.Application_PreInitialize();
             Application.NewApplication("", Application.WindowMode.Opaque);
-            NUIApplication.GetDefaultWindow();
```

Comment:
only this code removed.... ^^;
is this OK? doesn't this cause some behavior change?
I couldn't understand this patch, so I am asking. ^^;

Reply:
If Window is created in NUI, it will try to create LayoutController instance.
LayoutController makes Layout work so it should be created.

But in Preload time, LayoutController cannot be created because stage is not installed as huiyu mentioned in commit message.

So, if we remove this window creation attempt here, there will be no issue.

Reply:
I got it. Thank you for explanation~